### PR TITLE
Feat/onboard HvB & SocGen

### PR DESCRIPTION
**Pull request by nduchak**
# Description

# Contribution Checklist

- [ ] PR title starts with `(PE-<TICKET_NUMBER>)`
- [ ] Code approved
- [ ] Tests approved
- [ ] CI Tests pass

# Checklist

- [ ] Every contract variable/method declared as public/external private/internal
- [ ] Consider if this PR needs the `officeHours` modifier override
- [ ] Verify expiration (`30 days` unless otherwise specified)
- [ ] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [ ] Validate all addresses used are in Goerli changelog or known
- [ ] Notify any external teams affected by the spell so they have the opportunity to review
- [ ] Deploy spell to Goerli `ETH_GAS="XXX" ETH_GAS_PRICE="YYY" make deploy`
- [ ] Ensure contract is verified on `Goerli` etherscan
- [ ] Change test to use Goerli spell address and deploy timestamp
- [ ] Cast spell on Goerli `make spell="0x-deployed-spell-address" cast-spell`
- [ ] Run `make archive-spell` or `make date="YYYY-MM-DD" archive-spell` to make an archive directory and copy `Goerli-DssSpell.sol`, `Goerli-DssSpell.t.sol`, `Goerli-DssSpell.t.base.sol`, and `Goerli-DssSpellCollateralOnboarding.sol`
- [ ] `squash and merge` this PR

### CODE REVIEW

**Comment by nduchak** [comment]
[sc-832]

**Review by hbarcelos — CHANGES_REQUESTED** [review]
We should probably remove the `AT1` suffix from the official spell.

File: src/test/config.sol [hunk 1]
```diff
@@ -1671,5 +1669,35 @@ contract Config {
             calc_cut:     9900,
             lerp:         false
         });
+        afterSpell.collaterals["RWA009AT1-A"] = CollateralValues({
```
Comment (hbarcelos) [inline src/test/config.sol:1672]:
```suggestion
        afterSpell.collaterals["RWA009-A"] = CollateralValues({
```

**Review by hbarcelos — CHANGES_REQUESTED** [review]
There's still lots of references to `RWA009AT1` that should be changed to `RWA009`.

File: src/Goerli-DssSpell.t.sol [hunk 2]
```diff
@@ -223,31 +238,31 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // Insert new chainlog values tests here
-        // checkChainlogKey("CONTRACT_KEY");
+        // RWA009 
+        checkChainlogKey("RWA009AT1_A_JAR");
+        checkChainlogKey("RWA009AT1");
+        checkChainlogKey("MCD_JOIN_RWA009AT1_A");
```
Comment (dbkbali) [inline src/Goerli-DssSpell.t.sol:244]: Naz assuming this will be adjusted when you address @hbarcelos comment above

**Review by dbkbali — CHANGES_REQUESTED** [review]
Good work Naz will rereview when you push changes

**Review by hbarcelos — APPROVED** [review]
LGTM

File: src/Goerli-DssSpellCollateralOnboarding.sol [hunk 3]
```diff
@@ -33,57 +93,138 @@ contract DssSpellCollateralOnboardingAction {
...
+    uint256 public constant WAD               = 10**18;
+    uint256 public constant RAY               = 10**27;
+    uint256 public constant RAD               = 10**45;
+
+    // MIP21 components
+    address constant RWA009                   = 0xe4dEaE67031E6f159cE151be7ECBA0B50Dba66Fe;
+    address constant MCD_JOIN_RWA009_A        = 0xDf9dA30e1a80cC3532DF60caBbf488732E018095;
+    address constant RWA009_A_URN             = 0x5cF0cd96E21c634c8FA46FfFFB855A6AAe4AE47C;
+    address constant RWA009_A_JAR             = 0x7AAf3F8d07eF898e6fc55D3B7C88cCCFeb0275fF;
+    address constant RWA009_A_OUTPUT_CONDUIT  = 0x7a3D23Dc73F7ead55399597aAE6e525b3DF95A88;
```
Comment (dbkbali) [inline src/Goerli-DssSpellCollateralOnboarding.sol:110]: Noting that this address is not Genesis Public address - maybe we should add a comment here so we dont forget to change it on mainnet

**Review by dbkbali — APPROVED** [review]
Great work @nduchak

**Review by hbarcelos — APPROVED** [review]

File: src/Goerli-DssSpellCollateral.sol [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+        address MCD_VAT                           = CHANGELOG.getAddress("MCD_VAT");
+        address MCD_JUG                           = CHANGELOG.getAddress("MCD_JUG");
+        address MCD_SPOT                          = CHANGELOG.getAddress("MCD_SPOT");
+        // --------------------------- RWA Collateral onboarding ---------------------------
+        
+        // Onboard HvB
+        onboardRwa009(CHANGELOG, REGISTRY, MIP21_LIQUIDATION_ORACLE, MCD_VAT, MCD_JUG, MCD_SPOT);
+        
+        // Onboard SocGen
+        onboardRwa008(CHANGELOG, REGISTRY, MIP21_LIQUIDATION_ORACLE, MCD_VAT, MCD_JUG, MCD_SPOT);
```
Comment (hbarcelos) [inline src/Goerli-DssSpellCollateral.sol:396]: As we talked about it in earlier today, maybe we should keep the logical order here and onboard `RWA008` before `RWA009`.
Don't forget to update the tests (:

**Review by hbarcelos — COMMENTED** [review]
LGTM overall, just some small changes that we've discussed on the call earlier today.

**Pull request merged**